src/components/inline/DonationProjectCard/DonationProjectCard.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import DonationProjectCard from "./DonationProjectCard";
+import background from "./blackbackground.jpg";
+
+export default {
+  title: "Inline/DonationProjectCard",
+  component: DonationProjectCard,
+};
+
+const Template = () => (
+  <div style={{ backgroundImage: `url(${background})` }}>
+    <DonationProjectCard title="Star Hotel SRO" subtitle="280 People" />
+  </div>
+);
+
+export const DefaultDonationProjectCard = Template.bind({});
+
+DefaultDonationProjectCard.args = {
+  data: {
+    title: "Star Hotel SRO",
+    subtitle: "280 People",
```

Comment:
This is just my style but I think subtitle should be camel cased. @richardxia What do you think?

Reply:
Like subTitle?

Reply:
Yup

Reply:
I will wait for Richard's response to this.

Reply:
I would leave it as `subtitle`, since that's a full English word. I'd usually only camelcase something if it were two distinct words being combined into a single variable name. I might use camelcase for compound words in certain situations, but "subtitle" is not a compound word ("sub" is not a standalone word here), so this wouldn't apply here.

Reply:
Ok, keeping it as "subtitle."